pkg/virt-operator/creation/components/deployments.go

```diff
@@ -335,6 +335,7 @@ func NewControllerDeployment(namespace string, repository string, imagePrefix st
 	}
 
 	attachCertificateSecret(pod, VirtControllerCertSecretName, "/etc/virt-controller/certificates")
+
```

Comment:
super minor, there's a random line added here to a file that otherwise has no changes.